### PR TITLE
[ZEPPELIN-6001] k8s images fix

### PR DESCRIPTION
**Pull request by th1nksnow**
### What is this PR for?
Fixed some errors while building k8s images.

zeppelin-interpreter Dockerfile:
fixed JAVA_HOME env

zeppelin-server Dockerfile:
Error "bzip2: Compressed file ends unexpectedly;" fixed by adding ---no-iri flag to wget

changed ARG version in both Dockerfiles

### What type of PR is it?
Bug Fix

### CODE REVIEW

File: scripts/docker/zeppelin-interpreter/Dockerfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ ENV VERSION="${version}" \
 RUN set -ex && \
     /usr/bin/apt-get update && \
     DEBIAN_FRONTEND=noninteractive /usr/bin/apt-get install -y openjdk-11-jre-headless wget tini bzip2 && \
-    /usr/bin/wget -qO- https://micromamba.snakepit.net/api/micromamba/linux-64/latest | tar -xvj bin/micromamba && \
+    /usr/bin/wget --no-iri -qO- https://micromamba.snakepit.net/api/micromamba/linux-64/latest | tar -xvj bin/micromamba && \
```
Comment (Reamer) [inline scripts/docker/zeppelin-interpreter/Dockerfile:33]: How to reproduce the error you are getting?

**Review by Reamer — CHANGES_REQUESTED** [review]
A small question.

File: scripts/docker/zeppelin-interpreter/Dockerfile (hunk 1 above)
Comment (th1nksnow) [reply scripts/docker/zeppelin-interpreter/Dockerfile:33]: In my case i just run ```docker build -t zeppelin-interpreter:0.12.0-SNAPSHOT -f ./Dockerfile .``` on Docker Engine 25.0.3.
I found the solution [here](https://github.com/mamba-org/mamba/issues/2998#issuecomment-1829292600).

Comment (Reamer) [reply scripts/docker/zeppelin-interpreter/Dockerfile:33]: Very strange error. I can also reproduce this within Docker.

Comment (Reamer) [inline scripts/docker/zeppelin-interpreter/Dockerfile:33]: I would use the following setting of wget. The other option seems to me to switch off too much of the standard.
```suggestion
    /usr/bin/wget --remote-encoding=utf-8 -qO- https://micromamba.snakepit.net/api/micromamba/linux-64/latest | tar -xvj bin/micromamba && \
```

Comment (th1nksnow) [reply scripts/docker/zeppelin-interpreter/Dockerfile:33]: Sure, build is successful.

**Review by Reamer — APPROVED** [review]
LGTM

**Pull request merged**